uint64 timestamp		# time since system start (microseconds)

# NAOR DEBUG DATA

float32 land_target_x
float32 land_target_y

float32 velocity_setpoint_command_right
float32 velocity_setpoint_command_forward

float32 current_velocity_right
float32 current_velocity_forward

float32 openloop_command_right
float32 openloop_command_forward
